src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -16,6 +16,27 @@ public Rectangle(Integer width, Integer length, String color) {
         this.color = color;
     }
 
+    @Override
+    public int hashCode() {
+        return 31 * color.hashCode() * this.getWidth() * this.getLength();
```

Comment:
what will happen if getWidth() or getLength() == 0?
and try to use hashCode() to all fields

Reply:
???

Reply:
what about this comment?